Tiltfile

```diff
@@ -1,8 +1,38 @@
 load('ext://restart_process', 'docker_build_with_restart')
+load('ext://helm_resource', "helm_resource", 'helm_repo')
 
 # A list of directories where changes trigger a hot-reload of the sequencer
 hot_reload_dirs = ['app', 'cmd', 'tools', 'x']
 
+# Create localnet config file from defaults, and if default configuration doesn't exist in it - populate with default values
+localnet_config_path = "localnet_config.yaml"
+localnet_config_defaults = {
+    "relayers": {"count": 1},
+    "gateways": {"count": 0},
+    "helm_chart_local_repo": {"enabled": False, "path": "../helm-charts"},
```

Comment:
@Olshansk by default, this makes Tiltfile use the [helm repo](https://pokt-network.github.com/helm-charts), instead of git repository cloned on laptop. You can see this logic below.

Reply:
Not a blocker but just a `question`: why not have it be a git submodule and point to the local path?

Reply:
@Olshansk, I'd prefer not to manage submodule SHAs because they can complicate matters. In our current setup, developers don't need to worry about local updates; Helm handles this through its native repositories. It's important that we maintain this support for the community. Consider this a personal preference on my part—I aim to avoid using submodules whenever an alternative exists.

Reply:
Makes sense to me. PR is still approved